thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/util/ThirdEyeUtils.java

```diff
@@ -705,6 +706,16 @@ public static long getCachingPeriodLookback(TimeGranularity granularity) {
     return period;
   }
 
+  /**
+   * Check if the anomaly is detected by multiple components
+   * @param anomaly the anomaly
+   * @return if the anomaly is detected by multiple components
+   */
+  public static boolean isDetectedByMultipleComponents(MergedAnomalyResultDTO anomaly) {
+    String componentName = anomaly.getProperties().getOrDefault(PROP_DETECTOR_COMPONENT_NAME, "");
+    return componentName.contains(",");
```

Comment:
Convert "," to a static final string PROP_DETECTOR_COMPONENT_NAME_DELIMETER and use this for all the places it is used. E.g, combineComponents()/

Reply:
good point. updated